tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -3157,43 +3157,35 @@ def load(cls, path: str, verbosity: int = None, require_version_match: bool = Tr
 
         try:
             from ..version import __version__
-            version_load = __version__
+            version_current = __version__
```

Comment:
The existing naming was a bit confusing to me: I would expect `version_load` to correspond to the version of the loaded predictor, but in the old code it referred to the version of AG installed in the system. I changed the names `version_load` -> `version_current` and `version_init` -> `version_saved` to address this, but if you prefer the old naming, I will undo the change.

Reply:
LGTM